testrunner/execute.go

```diff
@@ -230,6 +232,16 @@ func removeObsoleteParentTests(tests []testResult) []testResult {
 	return results
 }
 
+func formatTestNames(tests []testResult) []testResult {
+	var out []testResult
+	for _, test := range tests {
+		re := regexp.MustCompile("[/_]")
+		test.Name = re.ReplaceAllString(test.Name, " ")
```

Comment:
According to https://github.com/exercism/go-test-runner/issues/77#issuecomment-1186284367, the fix were would be to:

> add a space after the slash and replace all the underscores with spaces

This code replaces slashes `/` with spaces instead of adding a space after the slash.

As an alternative to regexes, consider [strings.Replacer](https://pkg.go.dev/strings#NewReplacer), which allows specifying several string replacements once, and then applying those replacements to several strings by calling `Replace` on each one.